src/pages/upgrades/get-involved/index.js

```diff
@@ -244,24 +244,89 @@ const GetInvolvedPage = ({ data, location }) => {
     setClients(randomizedClients)
   }, [data])
 
+  const consensusClients = [
```

Comment:
This should be deleted - looks like a duplicate array?

Reply:
@corwintines @wackerow please clean up all the console warnings while we're at it:
```
warning /Users/samrichards/code/ef/ethereum-org-website/src/components/Layout.js
  96:6  warning  React Hook useEffect has a missing dependency: 'props.location.hash'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

/Users/samrichards/code/ef/ethereum-org-website/src/components/Morpher.js
  152:6  warning  React Hook useEffect has missing dependencies: 'morphTimeout', 'morpher', 'state.text', and 'state.words'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

/Users/samrichards/code/ef/ethereum-org-website/src/components/YouTube.js
  24:7  warning  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title

/Users/samrichards/code/ef/ethereum-org-website/src/pages/developers/learning-tools.js
  87:6  warning  React Hook useEffect has a missing dependency: 'sandboxes'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

/Users/samrichards/code/ef/ethereum-org-website/src/pages/upgrades/get-involved/index.js
  247:9  warning  'consensusClients' is assigned a value but never used  no-unused-vars
```

Reply:
Already put up a separate PR (#5097) for this cleanup =)